### PR TITLE
💄 #287 Clinician Invite Page: Add Link to Actual Assent Form File (PDF)

### PR DESCRIPTION
**Pull request by shailikadakia-oicr**
Converted assent form to a pdf, updated the assets dictionary for fr and en, and updated the link in clinician invite form to open assent form

#### Consent - UI
Added `assentFormPdf` to the assetUrls dictionary. This is imported into the `inviteFormText Dictionary` which adds to the text dictionary in `ClinicianInviteFormComponent`. This allows fr and en users to view the assent form in their respective languages.

Users can now view the assent form and download it. 

## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] [Axe DevTools](https://www.deque.com/axe/browser-extensions/) "Full Page Scan" of updated pages passes without introducing new Critical or Serious issues (UI changes only)
- [x] PR feedback has been addressed **if applicable**
- [ ] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files

### CODE REVIEW

File: apps/consent-ui/src/i18n/locales/en/assetUrls.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@
 
 const dictionary = {
 	studyConsentPdf: 'OHCRN_Main_Consent_Form-REB-clean_v2_5Jan2024.pdf',
+	assentFormPdf: 'OHCRN_Assent-v2_14Dec2023_clean.pdf',
```
Comment (samrichca) [inline apps/consent-ui/src/i18n/locales/en/assetUrls.ts:22]: we're keeping dictionaries alphabetized to make sure english & french dictionaries match. I use this plugin here https://marketplace.visualstudio.com/items?itemName=ue.alphabetical-sorter

Comment (shailikadakia-oicr) [reply apps/consent-ui/src/i18n/locales/en/assetUrls.ts:21]: Thank you Sam! I've downloaded the plug-in and pushed the changes :)

File: apps/consent-ui/src/components/views/Invite/ClinicianInviteForm/index.tsx [hunk 2]
```diff
@@ -274,7 +276,7 @@ const ClinicianInviteFormComponent = ({
 								<>
 									<p className={styles.assentFormAcknowledged}>
 										{textDict.assentFormAcknowledgedText1}{' '}
-										<Link href="" prefetch={false}>
+										<Link href={assentFormPdfUrl} prefetch={false}>
```
Comment (mistryrn) [inline apps/consent-ui/src/components/views/Invite/ClinicianInviteForm/index.tsx:279]: Let's make this PDF open in a few tab so the Clinician doesn't lose anything they may have already entered into the invite form when they click this link:
```suggestion
										<Link href={assentFormPdfUrl} prefetch={false} target="_blank">
```

**Review by mistryrn — CHANGES_REQUESTED** [review]
Looks great! One smol change request, other than that this should be good to go 🙂

File: apps/consent-ui/src/components/views/Invite/ClinicianInviteForm/index.tsx (hunk 2 above)
Comment (samrichca) [reply apps/consent-ui/src/components/views/Invite/ClinicianInviteForm/index.tsx:279]: thanks for pointing this out. we have it on another PDF link as well.

**Review by mistryrn — APPROVED** [review]
Looks good! Thanks for getting that change in 🙂

**Pull request merged**